devices/philips.js

```diff
@@ -700,6 +700,13 @@ module.exports = [
         description: 'Hue Flourish white and color ambiance table light with Bluetooth',
         extend: hueExtend.light_onoff_brightness_colortemp_color({colorTempRange: [153, 500]}),
     },
+    {
+        zigbeeModel: ['929003052601'],
+        model: '929003052601',
+        vendor: 'Philips',
+        description: 'Hue Flourish white and color ambiance table light with Bluetooth',
+        extend: hueExtend.light_onoff_brightness_colortemp_color({colorTempRange: [153, 500]}),
```

Comment:
TBH I am not sure about this color temp range. This device has a std Hue E26 bulb. I looked around and can't find any specs for the color temp range & there's nothing in the zigbee2mqtt logs. I added as the specs require it.

Reply:
Read this to figure out the color temp range: https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#_3-adding-converter-s-for-your-device